apps/designer/app/designer/features/style-panel/controls/select/select-control.tsx

```diff
@@ -21,7 +32,15 @@ export const SelectControl = ({
   const setValue = setProperty(styleConfig.property);
 
   return (
-    <Grid columns={2}>
+    <Grid
+      columns={2}
+      css={{
+        gridTemplateColumns: cssVars.use(
+          gridTemplateColumnsVar,
+          "repeat(2, 1fr)"
```

Comment:
use the fallback value, this is te same value that we get in Grid component from columns=2. I don't like the copy-paste here, since if we change columns to 3 this will not be very obvious.

Another option would be to introduce the CSS var straight in the Grid component and then reexport it from select control via that high level function

Reply:
removed columns prop, now we don't have duplication of the default value